app/src/main/java/com/example/caregiver/App.java

```diff
@@ -1,13 +1,23 @@
 package com.example.caregiver;
 
 import android.app.Application;
+
+import com.example.caregiver.model.Task;
 import com.kontakt.sdk.android.common.KontaktSDK;
 
+import java.util.List;
+
 public class App extends Application {
     @Override
     public void onCreate() {
         super.onCreate();
         KontaktSDK.initialize(this);
     }
+
+    /* Return a list of tasks associated with that caregivee */
+    public interface TaskCallback {
+        void onDataGot(List<Task> tasks);
```

Comment:
Can we reword this function to be "onDataReceived"?

Reply:
Fixed!